recipes/default.rb

```diff
@@ -250,14 +250,17 @@
   group librenms_group
   mode '0644'
   variables(
-    db_pass:  node['mariadb']['user_librenms']['password'],
-    user:     librenms_username,
-    path:     librenms_homedir,
-    auto_up:  node['librenms']['auto_update_enabled'],
-    xdp:      node['librenms']['autodiscover']['xdp'],
-    ospf:     node['librenms']['autodiscover']['ospf'],
-    bgp:      node['librenms']['autodiscover']['bgp'],
-    snmpscan: node['librenms']['autodiscover']['snmpscan'],
+<<<<<<< HEAD
```

Comment:
You have conflict markers in your file, you should fix that.

Reply:
Fixed.